### ISSUE TITLE
usage ?

### ISSUE THREAD

**Issue by fullflash** [JeremyFunk/hidefiles#3]
hi
i have created hide-files.json
and inserted needed folders
then what should we do in order to extension work ?
reloaded vscode

```
{
    "profiles": [
        {
            "name": "Default",
            "detail": "Hides huge, annoying folders",
            "hidden": [
                "node_modules/",
                ".vscode/",
                ".storybook/",
                ".github/",
                "android/",
                "ios/",
                "macos/",
                "web/",
                ".packages",
                ".metadata",
                "*.iml",
                "README.md"
            ]
        },
        {
            "name": "Strict",
            "detail": "Hide everything not immediately required",
            "hidden": [
                "$Default",
               ......
            ]
        }
    ]
}
```

**Comment by JeremyFunk** [comment]
It should work by simply running _HideFile: Reload Configuration_.
For Windows: CTRL+Shift+P -> _HideFiles: Reload Configuration_ -> Select Profile -> Press Enter.

![image](https://user-images.githubusercontent.com/29690247/140790423-f990bf61-a1f3-4a37-985d-7998928166d3.png)

![image](https://user-images.githubusercontent.com/29690247/140790484-78160f67-4e9c-4c62-8af6-5216bd1edadc.png)

Please let me know if it worked. Could you include your full hide-files.json, aswell as a screenshot of the project structure if it doesn't work? Would like to test it and fix the bug if there is one.

**Comment by JeremyFunk** [comment]
Closed due to inactivity. Likely cause was discussed in #4. Fixes and improvements in messaging to the user will be available with version 1.0.0.
